index.html

```diff
@@ -82,7 +82,7 @@ <h5>&quot;Karibu Kisumo!!!&quot;</h5>
         <h1>The 9th AfriCities Summit 2022</h1>
```

Comment:
- [ ] as your are allowed to have a different content from the given design, the project require that you stick to color and font size, Kindly fix the color and font size of you `h1` tag.
  your design :point_down: 
![Screenshot from 2022-08-02 22-37-43](https://user-images.githubusercontent.com/66097371/182478502-939013e3-f9df-47b7-a08c-cda8e268f475.png)
project design :point_down: 
![Screenshot from 2022-08-02 22-42-10](https://user-images.githubusercontent.com/66097371/182478660-8005adfd-b988-4f83-ac09-0ceab24b4705.png)